src/script/components/Conversation/ConversationCells/ConversationCells.tsx

```diff
@@ -66,9 +75,14 @@ export const ConversationCells = ({
   const isLoading = nodesStatus === 'loading';
   const isError = nodesStatus === 'error';
   const isSuccess = nodesStatus === 'success';
+
   const hasNodes = !!nodes.length;
+  const emptyView = !isError && !hasNodes && isCellsStateReady;
 
-  const emptyView = !isError && !hasNodes;
+  const isTableVisible = (isSuccess || isLoading) && isCellsStateReady;
+  const isLoadingVisible = isLoading && isCellsStateReady;
+  const isNoNodesVisible = !isLoading && emptyView && isCellsStateReady;
```

Comment:
this check is taken care of in `emptyView` 
```suggestion
  const isNoNodesVisible = !isLoading && emptyView;
```